code/bignum/AUTHORS.md

```diff
@@ -0,0 +1 @@
+This code was primarily written by Marina Polubelova (INRIA).
```

Comment:
you may want to add something along the lines of "Jonathan Protzenko contributed the design of the type classes hierarchy" so that if there's questions about that you can deflect and not be bothered

Reply:
Sure! Thanks once again for your help! :)